pkg/globalnet/controllers/ipam/node_handler.go

```diff
@@ -0,0 +1,105 @@
+package ipam
+
+import (
+	"fmt"
+
+	"github.com/submariner-io/admiral/pkg/log"
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog"
+
+	"github.com/submariner-io/submariner/pkg/routeagent_driver/constants"
+)
+
```

Comment:
So these functions were moved to a new file - was this just a cleanup/refactor or were they actually changed?

Reply:
Thanks for reviewing @tpantelis.

It's pretty much the same. Except that while calling the API syncNodeRules(), we are now passing node.Name